README.md

```diff
@@ -186,7 +186,7 @@ export function SmartEdgeWithButtonLabel(props) {
 
 ## Advanced Custom Smart Edges
 
-The `getSmartEdge` function also accepts an optional object `options`, which let's you configure aspects of the path-finding algorithm. You may use it like so:
+The `getSmartEdge` function also accepts an optional object `options`, which allows you configure aspects of the path-finding algorithm. You may use it like so:
```

Comment:
typo: Missing a "to": "which allows you **to** configure"

Reply:
Thank you @tisoap , I have updated the README now.